--- win-linux/src/cthemes.cpp.orig	2025-10-15 19:44:51.152172963 +0800
+++ win-linux/src/cthemes.cpp	2025-10-15 20:04:05.591866724 +0800
@@ -1,10 +1,5 @@
 
 #include "cthemes.h"
-#ifdef Q_OS_LINUX
-# include <gtk/gtk.h>
-# include <gio/gio.h>
-# include <glib.h>
-#endif
 #include "defines.h"
 #include "utils.h"
 #include <QSettings>
@@ -19,6 +14,13 @@
 #include <QApplication>
 #include <QJsonArray>
 
+#undef signals
+#ifdef Q_OS_LINUX
+# include <gtk/gtk.h>
+# include <gio/gio.h>
+# include <glib.h>
+#endif
+
 #define QSTRING_FROM_WSTR(s) QString::fromStdWString(s)
 #define REGISTRY_THEME_KEY "UITheme"
 // #define REGISTRY_THEME_KEY_7_2 "UITheme2"
